tests/test_survey_schema_translation.py

```diff
@@ -577,3 +577,469 @@ def test_checkbox_null_label():
+                                                        "source": "answers",
+                                                        "identifier": "registration-date",
+                                                    },
+                                                    "second_date": {"value": "now"},
+                                                },
+                                            }
+                                        ],
+                                    },
+                                ],
+                            },
+                            "answers": [
+                                {
+                                    "type": "Radio",
+                                    "label": "WELSH - Has this company been trading in the UK? (Mandatory)",
+                                    "mandatory": True,
+                                    "options": [
+                                        {"label": "WELSH - Yes", "value": "Yes"},
+                                        {"label": "WELSH - No", "value": "No"},
+                                    ],
+                                }
+                            ],
+                        },
+                    },
+                ],
+                "edit_block": {
+                    "type": "ListEditQuestion",
+                    "question": {
+                        "type": "General",
+                        "title": "WELSH - What is the name and registration number of the company?",
+                        "answers": [
+                            {
+                                "label": "WELSH - Name of UK company or branch (Mandatory)",
+                                "mandatory": True,
+                                "type": "TextField",
+                            }
+                        ],
+                    },
+                },
+                "remove_block": {
+                    "type": "ListRemoveQuestion",
+                    "question": {
+                        "type": "General",
+                        "title": "WELSH - Are you sure you want to remove this company or UK branch?",
+                        "answers": [
+                            {
+                                "type": "Radio",
+                                "options": [
+                                    {
+                                        "label": "WELSH - Yes",
+                                        "value": "Yes",
+                                        "action": {"type": "RemoveListItemAndAnswers"},
+                                    },
+                                    {"label": "WELSH - No", "value": "No"},
+                                ],
+                            }
+                        ],
+                    },
+                },
+            },
+        ],
+        "language": "cy",
+    }
+
+    assert expected == translated.schema
+
+
+def test_content_variants_translate():
```

Comment:
Can we add a normal list collector content block test (without variants) just to be safe and prove it gets added to extractable strings?

Reply:
The test above this is a list collector without variants? It just has repeating blocks

Reply:
I mean, list collector content block without content variants, just contents. I've checked in validator and list collector content block can have one of two elements:
```
"oneOf": [
      {
        "required": ["content"]
      },
      {
        "required": ["content_variants"]
      }
    ]
```

Reply:
It doesn't seem like you test "ListCollectorContent" block type, so it would probably need tests both one with `content` and `content_variants`.

Reply:
Tests added